airflow/providers/cncf/kubernetes/operators/kubernetes_pod.py

```diff
@@ -348,15 +351,17 @@ def launcher(self) -> pod_launcher.PodLauncher:
         return pod_launcher.PodLauncher(kube_client=self.client)
 
     @cached_property
-    def client(self) -> CoreV1Api:
-        # todo: use airflow Connection / hook to authenticate to the cluster
-        kwargs: Dict[str, Any] = dict(
+    def k8s_hook(self):
```

Comment:
```suggestion
    def hook(self):
```

nit, for consistency with other operators?